### PR TITLE
Readme install

### PR DESCRIPTION
**Pull request by thadk**
This PR clarifies a few things in the readme and also adds a [tasks.json](https://code.visualstudio.com/Docs/editor/tasks), which I'm still testing, that enables you to run all the docker-compose sessions for DocumentCloud at once using `Cmd-Shift-B` or `Ctrl-Shift-B` (win/linux) in VS Code and probably Github Codingspaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,15 @@ User account service for MuckRock and DocumentCloud
...
 3. Run the dotenv initialization script - `python initialize_dotenvs.py`
 This will create files with the environment variables needed to run the development environment.
-4. Start the docker images - `inv up`
-This will build and start all of the docker images using docker-compose.  It will bind to port 80 on localhost, so you must not have anything else running on port 80. The invoke tasks specify the `local.yml` configuration file for docker-compose.  If you would like to run docker-compose commands directly, set the environment variable `export COMPOSE_FILE=local.yml`.
-5. Set `dev.squarelet.com` to point to localhost - `sudo echo "127.0.0.1   dev.squarelet.com" >> /etc/hosts`
-6. Enter `dev.squarelet.com` into your browser - you should see the Squarelet home page.
-
+4. You may need to provide valid testing values for `STRIPE_PUB_KEYS`, `STRIPE_SECRET_KEYS` and set `STRIPE_WEBHOOK_SECRETS=None` from the MuckRock team (multiple values are comma separated only, no square braces) 
+      - You must always fully `docker-compose down` or Ctrl-C each time you change a `.django` file of a docker-compose session for it to take effect (as far as I know).
```
Comment (mitchelljkotler) [inline README.md:21]: I would say for any `.env` file.  I would also remove the weasel words - 
> You will need to provide
Remove as far as I know - you do need to bring the docker container down and up again for `.env` changes to take affect,

File: README.md [hunk 2]
```diff
@@ -17,11 +17,15 @@ User account service for MuckRock and DocumentCloud
...
+      - Avoid changing Squarelet's `.django` file frequently to prevent Docker network problems from `docker-compose down`.
```
Comment (mitchelljkotler) [inline README.md:22]: I would remove this - once you are set up there is no reason to change `.env` files often, but you certainly can update it as much as needed - you just need to ensure the squarelet network is up before starting DocumentCloud.

File: README.md [hunk 3]
```diff
@@ -17,11 +17,15 @@ User account service for MuckRock and DocumentCloud
...
+5. Set the environment variable `export COMPOSE_FILE=local.yml` in each of your command lines.
+5. Start the docker images - `inv up`
+This will build and start all of the Squarelet session docker images using docker-compose.  It will bind to port 80 on localhost, so you must not have anything else running on port 80. The "invoke" tasks from `tasks.py` specify the `local.yml` configuration file for docker-compose.
+6. Set `dev.squarelet.local` and `dev.squarelet.com` to point to localhost - `sudo echo "127.0.0.1   dev.squarelet.com dev.squarelet.local" >> /etc/hosts`
```
Comment (mitchelljkotler) [inline README.md:26]: I think we shoul dprobably just use `dev.squarelet.local` and remove references to `dev.squarelet.com`

**Review by mitchelljkotler — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (thadk) [reply README.md:22]: Removing this but ultimately decided that my trouble with networks didn't have to do with my docker version. It happened even with latest docker, not sure why.

File: README.md (hunk 1 above)
Comment (thadk) [reply README.md:21]: Fixed

Comment (thadk) [reply README.md:21]: But not quite sure what you meant by "for any .env file" — I think these values are only needed for Squarelet, not DocumentCloud's .django files?

File: README.md (hunk 3 above)
Comment (thadk) [reply README.md:25]: Do you want me to remove `dev.squarelet.com` from the Django config files as well?
